cmd/influxd/launcher/launcher.go

```diff
@@ -30,6 +30,7 @@ import (
 	"github.com/influxdata/influxdb/kit/prom"
 	"github.com/influxdata/influxdb/kit/signals"
 	"github.com/influxdata/influxdb/kit/tracing"
+	kithttp "github.com/influxdata/influxdb/kit/transport/http"
```

Comment:
nit picky, should this be kitHTTP?

Reply:
with pkgs, its not pretty typical to have it be all lowercase